policyengine_us/variables/gov/states/ms/tax/income/taxable_income/ms_taxable_income_joint_head_or_spouse.py

```diff
@@ -1,10 +1,10 @@
 from policyengine_us.model_api import *
 
 
-class ms_taxable_income_joint(Variable):
+class ms_taxable_income_joint_head_or_spouse(Variable):
```

Comment:
```suggestion
class ms_unadjusted_taxable_income_joint(Variable):
```

Reply:
I think we dont need the head and spouse condition here as we already assign all income from the dependents in the ms_agi file